BangazonAPI/BangazonAPI/appsettings.json

```diff
@@ -6,6 +6,6 @@
   },
   "AllowedHosts": "*",
   "ConnectionStrings": {
-    "DefaultConnection": "Server=localhost\\SQLEXPRESS03;Database=BangazonAPI;Trusted_Connection=True;"
+    "DefaultConnection": "Server=localhost\\SQLEXPRESS01;Database=BangazonAPI;Trusted_Connection=True;"
```

Comment:
Were you trying to add this to your `.gitignore`? I think it's a good idea, but it didn't add correctly because this is on your PR. Try untracking this file from git and then pushing back up. If you choose not to have git ignore it, is the rest of your team running on `SQLEXPRESS`? If so, I'd change it before you merge to be nice to them. If they're all running different instaces of sql server, it's a moot point and you'll all have to change it